projects/main/src/app/views/keys/key-backup-dialog/key-backup-dialog.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { STEPPER_GLOBAL_OPTIONS } from '@angular/cdk/stepper';
+
+@Component({
+  selector: 'app-key-backup-dialog',
+  templateUrl: './key-backup-dialog.component.html',
+  styleUrls: ['./key-backup-dialog.component.css'],
+  providers: [{
+    provide: STEPPER_GLOBAL_OPTIONS, useValue: { showError: true }
+  }]
+})
+export class KeyBackupDialogComponent implements OnInit {
+
+  saved: boolean = false;
+  checked: boolean = false;
+  inputMnemonic: string = "";
+
+  now = new Date();
+  sec = this.now.getSeconds();
+  requiredMnemonicNumber = this.sec % 12
+
+  constructor(
+    @Inject(MAT_DIALOG_DATA)
+    public readonly data: {
+      mnemonic: string,
+      privatekey: string
+    },
+    public matDialogRef: MatDialogRef<KeyBackupDialogComponent>,
+  ) { }
+
+  onClickOkButton(input: boolean): void {
+    // ボタンが押されたときは「checked」を呼び出し元に渡す。
+    //Todo:渡した「checked」を判断に使用する。
+    this.matDialogRef.close(this.checked);
```

Comment:
@taro04 cc: @KimuraYu45z @Senna46 
例えば以下のように、単なる1個のbooleanな値を返すのではなく、ファイルダウンロードしたか否かのsavedと、ニーモニックの確認がOKかNGかのcheckedそれぞれの値を独立して返してあげたほうが、柔軟で正確な制御ができるのでは？と思いました。

```suggestion
    this.matDialogRef.close({ saved: this.saved, checked: this.checked });
```